src/createServer.js

```diff
@@ -1,3 +1,74 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [pathName, queryString] = req.url.split('?');
+    const params = new URLSearchParams(queryString);
+    const toCase = params.get('toCase');
+    const textToConvert = pathName.slice(1);
+
+    const errors = [];
+
+    if (!textToConvert) {
+      errors.push({
+        // eslint-disable-next-line
+        message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message:
+          // eslint-disable-next-line
+          '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
move outside func `createServer` and rename to SUPPORTED_CASES

Reply:
Hi, Volodymyr. Moved `const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];`  outside function createServer.